docs/topics/gradle.md

```diff
@@ -975,16 +975,16 @@ _Kotlin compiler execution strategy_ defines where the Kotlin compiler is execut
+|----------------|--------------------------------------|-------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Daemon         | Inside its own daemon process        | Yes                     | *The default strategy*. Can be shared between different Gradle daemons.                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| In process     | Inside the Gradle daemon process     | No                      | May share the heap with the Gradle daemon. Note that the in process execution is *slower* than the Daemon execution strategy. This is so because Gradle spawns multiple [workers](https://docs.gradle.org/current/userguide/worker_api.html) with the Kotlin compiler, and they could be not yet [JIT-optimized](https://en.wikipedia.org/wiki/Just-in-time_compilation#:~:text=In%20computing%2C%20just%2Din%2D,time)%20rather%20than%20before%20execution.) when the compilation runs. Also, the Kotlin daemon has additional caches. |
+| Out of process | In a separate process for each call  | No                      | The slowest execution strategy.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
Imho, still better to explain the difference:

> The slowest execution strategy. Similar to 'in-process', but additionally creates a separate Java process within a Gradle worker to execute a Kotlin compiler.

Reply:
Ok, I'll add.